content/applications/general/email_communication/faq.rst

```diff
@@ -1,174 +1,185 @@
+and sometimes even per minute. This is the same for Odoo, which limits a customer's sending to
+prevent Odoo's e-mail servers from being blacklisted.
 
 Here are the default limits for new databases:
 
 - 200 emails/day for Odoo Online and Odoo.sh databases with an active subscription,
 
 - 50 emails/day for one-app free and trial databases,
 
-- in case of migration, your daily limit might be reset to 50 emails a day.
+- In case of migration, the database daily limit might be reset to 50 emails a day.
 
-In case you hit the limit, you can:
+In case the limit is reached, the following actions can be taken:
 
-- Ask our support team to increase your daily limit. We will analyze the situation of your database
-  depending on (non-exhaustive list):
+- Ask Odoo's support team to increase the daily limit. The Support Team will analyze the situation
+  of the database depending on (non-exhaustive list):
 
-  - How many users in your database,
+  - How many users in the database,
   - Which apps are installed,
-  - Your bounce rate: the percentage of email addresses that did not receive your emails because
-    it was returned by a mail server on its way to the final recipient. You can contact the `support
+  - The bounce rate: the percentage of email addresses that did not receive emails because they were
+    returned by a mail server on its way to the final recipient. Contact the `support
     <https://www.odoo.com/help>`_.
 
-- Use your own outgoing email server to be independent of Odoo’s mail limit (please refer
-  to :doc:`the corresponding documentation </applications/general/email_communication/email_servers>`),
-- Wait until 11pm UTC for the reset and click on the retry button: The :ref:`Developer mode <developer-mode>`
-  must be activated. Then, go to :menuselection:`Settings --> Technical --> Emails`
-
-.. image:: faq/email-retry-technical.png
-   :align: center
-   :alt: Retry button of an emails
+- Use a external outgoing email server to be independent of Odoo's mail limit (refer to :doc:`the
+  corresponding documentation </applications/general/email_communication/email_servers>`),
+- Wait until 11pm UTC for the reset and click on the retry button: The :ref:`Developer mode
+  <developer-mode>` must be activated. Then, go to :menuselection:`Settings --> Technical -->
+  Emails`
 
 .. warning::
-   The daily limit is global to your database and can rise quickly! By default an internal message,
-   a notification, a note, etc. counts as an email in your daily limit if it notifies someone.
+   The daily limit is global to the database and can rise quickly! By default an internal message,
+   a notification, a note, etc. counts as an email in the daily limit if it notifies someone.
 
-You can mitigate this by receiving your :ref:`notifications in Odoo <discuss_app/notification_preferences>`
-instead of by emails.
+This can be mitigated by receiving :ref:`notifications in Odoo
+<discuss_app/notification_preferences>` instead of by emails.
```

Comment:
```suggestion
This can be mitigated by receiving :ref:`notifications in Odoo
<discuss_app/notification_preferences>`, instead of emails.
```
cleaned up a bit, but please make sure that the meaning is still accurate with my suggested removal of the word "by"

Reply:
I've changed it to "by email". Thanks for cleaning up.

Reply:
Sounds better, thanks.